core/block/editor/table/table.go

```diff
@@ -467,7 +467,7 @@ func (t *Editor) cleanupTables(_ smartblock.ApplyInfo) error {
 		log.Errorf("cleanup iterate: %s", err)
 	}
 
-	if err = t.sb.Apply(s); err != nil {
+	if err = t.sb.Apply(s, smartblock.KeepInternalFlags); err != nil {
```

Comment:
why it's here?

Reply:
CleanupTables is the only hook that is executing on ObjectClose. Last command is triggered by client for some reason on ObjectSetObjectType. When we change type, we don't want to delete SelectTemplate flag, that's why I pass KeepInternalFlags to Apply here